include/picongpu/plugins/radiation/Radiation.hpp

```diff
@@ -387,6 +399,15 @@ class Radiation : public ISimulationPlugin
   {
     radiation->deviceToHost();
     __getTransactionEvent().waitForFinished();
+
+    auto dbox = radiation->getHostBuffer().getDataBox();
+    int numAmp = elements_amplitude();
+    // update the main result matrix (y index zero)
+    for( int resultIdx = 1; resultIdx < numJobs; ++resultIdx )
+        for( int ampIdx = 0; ampIdx < numAmp; ++ampIdx )
+        {
+            dbox(DataSpace<DIM2>( ampIdx, 0 ) ) += dbox(DataSpace< 2 >( ampIdx, resultIdx ) );
```

Comment:
Double use of `DIM2` and `2` for `DataSpace` template argument

Reply:
fixed